modules/auxiliary/scanner/ssh/libssh_auth_bypass.rb

```diff
@@ -50,6 +51,26 @@ def initialize(info = {})
     ])
   end
 
+  # Vulnerable since 0.6.0 and patched in 0.7.6 and 0.8.4
+  def check_banner(ip, version)
+    version =~ /libssh_([\d.]+)$/ && $1 && (v = Gem::Version.new($1))
+
+    if v.nil?
```

Comment:
this will still report a F+ when vendors backport without bumping version, but it cannot be helped